zendesk/sideload/ticket.go

```diff
@@ -0,0 +1,17 @@
+package sideload
+
+import "time"
+
+type TicketDates struct {
+	AssigneeUpdatedAt *time.Time `json:"assignee_updated_at"`
+	RequesterUpdatedAt *time.Time `json:"requester_updated_at"`
+	StatusUpdatedAt *time.Time `json:"status_updated_at"`
+	InitiallyAssignedAt *time.Time `json:"initially_assigned_at"`
+	AssignedAt *time.Time `json:"assigned_at"`
+	SolvedAt *time.Time `json:"solved_at"`
+	LatestCommentAddedAt *time.Time `json:"latest_comment_added_at"`
+}
+
+func IncludeTicketDates(dates *TicketDates) SideLoader {
```

Comment:
This func isn't strictly speaking necessary, but funcs like this might be a nice helper function.

Going to put it here for now as a potential talking point